spring-cloud-app-broker-core/src/main/java/org/springframework/cloud/appbroker/deployer/BackingSpaceManagementService.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.appbroker.deployer;
+
+import java.util.List;
+
+import reactor.core.publisher.Flux;
+
+public interface BackingSpaceManagementService {
+
+//	Flux<String> createTargetSpaces(List<String> targetSpaces);
+
+	Flux<String> deleteTargetSpaces(List<String> targetSpaces); // TODO: is it possible to have multiple target spaces?
```

Comment:
I guess if users don't specify targetSpec but set `target` directly in properties then we could end up with multiple spaces? I can remove the TODO if you are fine with passing in a list of spaces to delete.